### ISSUE TITLE
aniwave.to: ads

### ISSUE THREAD

**Issue by dutchiemapd** [uBlockOrigin/uAssets#20226]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] This is NOT a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://aniwave.to/watch/one-piece.ov8/ep-54`

### Category

ads

### Description

I was attempting to watch the anime One Piece on the streaming site aniwave.to, when suddenly an ad (see image) popped up, right in the middle of the screen. I didn't click it thankfully.

After trying a few more times the ad wouldn't go away, so i looked online on what to do. I then emptied all caches and updated now. I then tried again. It didn't work. It was at this point i decided to remove Ublock from Chrome and download it again. Sadly that didn't work either. I then tried restarting my laptop but once again it didn't work.

I decided to make a thread on the r/uBlockOrigin subreddit on the social media platform Reddit (https://www.reddit.com/r/uBlockOrigin/comments/17dwejl/comment/k5zbvli/?context=3)  where i told the same story as above. I then got redirected to this site to file a report. 

Please note that i am not very well informed in the world of adblockers and such, i dont understand a lot of tech mumbo jumbo. I would really appreciate it if you tried to explain it in simpler terms.

I hope i have been informative enough. If you have any questions or comments please let me know.

Thank you in advance. 

### Other extensions used

None

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

 
![sketchyad](https://github.com/uBlockOrigin/uAssets/assets/148709303/a50a6ec0-d091-4a17-bf34-e242fb9a38fc)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Chromium: 118
filterset (summary):
 network: 105831
 cosmetic: 42141
 scriptlet: 18881
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  NLD-0: 2701-10, 27m
  easylist: 72098-17, 27m
  easyprivacy: 33104-65, 27m
  plowe-0: 3727-1073, 27m
  ublock-badware: 7259-132, 27m
  ublock-filters: 35121-148, 27m
  ublock-privacy: 589-6, 27m
  ublock-quick-fixes: 243-93, 27m
  ublock-unbreak: 2132-33, 27m
  urlhaus-1: 11600-0, 27m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 869 ms
 maxAssetCacheWait: 36 ms
popupPanel:
 blocked: 11
 network:
  amung.us: 1
  bidgear.com: 4
  coldvain.com: 1
  disqus.com: 1
  googletagmanager.com: 1
  sharethis.com: 2
  superherosnout.com: 1
 extended:
  ##.adx
  ##+js(abort-on-property-read, AaDetector)
  ##+js(href-sanitizer, a[href^="https://disq.us/url?url="][title^…
  ##+js(href-sanitizer, a[href^="https://disq.us/?url=http"], ?url…
```
</details>

**Comment by mapx-** [comment]
@dutchiemapd You have to give all the necessary details: you should have said that it is about the MyCloud server.

**Comment by dutchiemapd** [comment]
@mapx- I'm sorry. I didn't think changing servers would make a difference. But changing servers actually helped...I didn't think it would be that simple.

I'm sorry for wasting everyone's time. If i have issues again i'll let you know.
